releng/tools/rmake

```diff
@@ -7,24 +7,38 @@ import multiprocessing
 from common import *
 
 usage="""Usage:
-    rmake --all                Compile all configured suites
-    rmake --list               Show available compilation targets
-    rmake --doc                Build & serve documentation (rebuilds on every change)
-    rmake --help               Print this help
-    rmake <targetName>...      Compile one or multiple targets
-    rmake <pathPrefix>...      Compile one or multiple targets in path
-    rmake <suitename>...       Compile given suite
-    rmake clean                Clean all suite builds
-    rmake clean-<suitename>... Clean given suite
+    rmake --all                  Compile all configured suites
+
+    rmake --list                 Show available compilation targets
+    rmake --doc                  Build & serve documentation (rebuilds on every change)
+    rmake --help                 Print this help
+
+    rmake <targetName>...        Compile one or multiple targets
+    rmake <pathPrefix>...        Compile one or multiple targets in path
+    rmake <suitename>...         Compile given suite
+
+    rmake --test --all           Compile and run all configured tests
+    rmake --test <targetName>... Compile and run test for target
```

Comment:
I'm not sure if I understand this correctly should you provide the test target name or do you provide a target name, which has some tests associated to it?
The wording for me implies the latter.

Reply:
Currently, it is a test-target name. It should be, as there is no direct mapping between targets and tests - I can have target A and test test-something1 and test-something2.

And I just realized that the tab completion isn't working as it is supposed to. I'll fix it.

Reply:
I would than change the `for target` in the `compile and run test for target` (it confused me more then it helped) - and maybe change `<targetName>` to `<testTargetName>`